src/pybind/mgr/mgr_module.py

```diff
@@ -40,6 +42,25 @@
     "creating",
     "unknown"]
 
+def is_ipv6_enabled():
```

Comment:
Much better. Thought, `ip` and `/proc` are linux-only. Is there an existing Python module available that would provide this  functionality also for BSD?

Reply:
The `netifaces` module might be the appropriate platform-independent solution

Reply:
what about ifconfig?

Reply:
> what about ifconfig?

``ifconfig`` is superseded by ``ip``. Debian for example does not install the ifconfig package anymore.